relay-event-normalization/src/normalize/processor.rs

```diff
@@ -959,13 +961,13 @@ fn remove_invalid_measurements(
             None => return false,
         };
 
-        if !is_valid_metric_name(name) {
+        let Some(name) = try_normalize_metric_name(name) else {
```

Comment:
If we get a changed name back here, we need to update the measurement.

Reply:
This is now a boolean check again as before